applications/ConstitutiveLawsApplication/tests/TractionSeparationLaw/traction_separation_law_test_materials.json

```diff
@@ -0,0 +1,80 @@
+{
+    "properties" : [{
+        "model_part_name" : "Structure.Parts_Solid_CubePart",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "TractionSeparationLaw3D", // TractionSeparationLaw3D
```

Comment:
Be careful with comments in `.json` files. While some libraries permit them, the standard does not allow them.

Reply:
Thanks for the comment. I modified the materialJson file.